docs/templates/TEMPLATE_ASSEMBLY_a-collection-of-modules.md

```diff
@@ -0,0 +1,27 @@
+# A collection of modules
+
+{% comment %}
+If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
+{% endcomment %}
+
+This paragraph is the assembly introduction. It explains what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. Can include more than one paragraph. Consider using the information from the user story.
+
+## Prerequisites
+
+* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
+* You can also link to other modules or assemblies the user must follow before starting this assembly.
+* Delete the section title and bullets if the assembly has no prerequisites.
+
+{% comment %}
+The following include statements pull in the module files that comprise the assembly. Include any combination of concept, procedure, or reference modules required to cover the user story. You can also include other assemblies.
+{% endcomment %}
+
+{% include_relative TEMPLATE_CONCEPT_concept_explanation.md %}
+
+{% include relative TEMPLATE_PROCEDURE_doing_one_procedure.md %}
+
+## Additional resources (or Next steps)
+
+* A bulleted list of links to other material closely related to the contents of the assembly, including xref links to other assemblies in your collection.
```

Comment:
`xref` is an adoc thing right? Is there a markdown equivalent?

Reply:
yeah, this is a simple oversight on my part. we should change those xref's to normal links. some of this will depend on how we decide to structure the more complex content, especially if we are linking back to docs we create.